porcupine/menubar.py

```diff
@@ -264,6 +263,40 @@ def update_enabledness(*junk: object) -> None:
     return update_enabledness
 
 
+def _generate_filetab_event(path: str) -> None:
+    event = f"<<FiletabCommand:{path}>>"
+    log.debug(f"Generating event on selected filetab: {event}")
+    tab = get_tab_manager().select()
+    assert isinstance(tab, tabs.FileTab)
+    tab.event_generate(event)
+
+
+def add_filetab_command(path: str) -> None:
```

Comment:
The new boilerplate seems to be to call this, and then create `on_new_filetab()` function just to bind to event. Change this to take optional function that will be called with filetab as argument.